tests/validation_suite/automation_framework/utilities/workflow.py

```diff
@@ -0,0 +1,128 @@
+# Copyright 2019 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pytest
+import logging
+import time
+import json
+
+from avalon_client_sdk.http_client.http_jrpc_client import HttpJrpcClient
+from error_code.error_status import WorkOrderStatus
+logger = logging.getLogger(__name__)
+
+
+def submit_request(uri_client, input_json_str1, output_json_file_name):
+    """ Function to submit request to enclave,
+    retrieve response and write files of input and output.
+    Input Parameters : uri_client, input_json_str1, output_json_file_name
+    Returns : response"""
+
+    import json
+    req_time = time.strftime("%Y%m%d_%H%M%S")
+    input_json_str = json.dumps(input_json_str1)
+    # write request to file
+    signed_input_file = ('./results/' + output_json_file_name + '_' + req_time
+                         + '_request.json')
+    with open(signed_input_file, 'w') as req_file:
+        req_file.write(json.dumps(input_json_str, ensure_ascii=False))
+        # json.dump(input_json_str1, req_file)
+
+    logger.info('**********Received Request*********\n%s\n', input_json_str)
+    # submit request and retrieve response
+    response = uri_client._postmsg(input_json_str)
+    logger.info('**********Received Response*********\n%s\n', response)
+
+    # write response to file
+    response_output_file = ('./results/' + output_json_file_name + '_'
+                            + req_time + '_response.json')
+    with open(response_output_file, 'w') as resp_file:
+        resp_file.write(json.dumps(response, ensure_ascii=False))
+
+    return response
+
+
+def is_valid_params(request_elements, keys_count=None):
+    """
+    Fucntion to check the number of parameters in submit requests.
```

Comment:
Is this function validating some params ? The function name and implementation doesn't match

Reply:
Done